scripts/version.sh

```diff
@@ -105,13 +105,8 @@ if [[ $update ]]; then
 
     # Update the Podspec version:
     sed -i '' -e "/^[[:space:]]*s\.version/ {s/\".*\"/\"$version\"/; }" ../BranchSDK.podspec
-
-    # Update the Carthage version:
-    plutil -replace CFBundleVersion -string "$version"  ../Branch-TestBed/Framework-Info.plist
-    plutil -replace CFBundleShortVersionString -string "$version"  ../Branch-TestBed/Framework-Info.plist
   
```

Comment:
@echo-branch These are updating `Framework-Info.Plist`. Next command is updating `BranchSDK.xcodeproj/project.pbxproj`  file.